cmd/virt-handler/virt-handler.go

```diff
@@ -393,10 +394,12 @@ func (app *virtHandlerApp) Run() {
 	se, exists, err := selinux.NewSELinux()
 	if err == nil && exists {
 		log.DefaultLogger().Infof("SELinux is reported as '%s'", se.Mode())
-		// Install KubeVirt's virt-launcher policy
-		err = se.InstallPolicy("/var/run/kubevirt")
-		if err != nil {
-			panic(fmt.Errorf("failed to install virt-launcher selinux policy: %v", err))
+		if !app.noCustomSELinuxPolicy {
```

Comment:
Why don't you just inspect the clusterConfig here to see if the feature gate is enabled? It would save you a lot of code.

Also with your approach right now the feature gate is honored only when you install/upgrade/downgrade kubevirt but it gets ignored if you just define the feature gate and restart virt-handler.

Reply:
Probably worth following this pattern https://github.com/kubevirt/kubevirt/blob/main/cmd/virt-handler/virt-handler.go#L304

Reply:
Yes, that would solve it, but why jump through hoops (defining a cmdline arguments, passing it from an install strategy, etc..) when we can simply check the FG inside virt-handler?

Reply:
Good point, for some reason I was convinced we didn't have access to the clusterConfig at that stage...
@xpivarc do you still think I should follow the patern you mentioned, or is it ok to just inspect the clusterConfig?
Thanks for the reviews!

Reply:
Actually there was a reason for changing virt-handlers arguments! The idea was to have it auto-restart when the feature gate is changed.
If I check the feature gate directly inside of virt-handler, then I need a callback, like Lubo suggested, otherwise I'll miss changes.

Reply:
If you want to cache the value of the FG I'd just check the FG during virt-handler init and then use the callback like Lubo suggested, If you implement it like this it's definitely simpler and shorter.

Reply:
Done!
@acardace @xpivarc PTAL!